build-ipa.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -ex
+
+TMPDIR=$(mktemp -d -p /var/tmp)
+TMPDIR_RAMDISK=$(mktemp -d -p /var/tmp)
+
+chmod 755 $TMPDIR
+cd $TMPDIR
+tar -xf /usr/share/rhosp-director-images/ironic-python-agent-latest.tar
+cd $TMPDIR_RAMDISK
+/usr/lib/dracut/skipcpio $TMPDIR/ironic-python-agent.initramfs | zcat | cpio -ivd
+# NOTE(elfosardo) we could inject a list of packages that we want to add, based
+# on what we download in the main image and call this part only if we actually
+# have packages in the list.
+# Also version tagging is something we should consider.
+# And cookies.
+rpm2cpio /tmp/packages/openstack-ironic-python-agent*.rpm | cpio -ivdum
+rpm2cpio /tmp/packages/python3-ironic-python-agent*.rpm | cpio -ivdum
+rpm2cpio /tmp/packages/python3-ironic-lib*.rpm | cpio -ivdum
```

Comment:
What am I missing, we're adding files from packages that are already in the ramdisk?

Reply:
these are not the same packages in the ramdisk
since the ramdisk uses old versions of the same packages, we download the new version in the image and inject the new content in the ramdisk